plonky2/src/fri/batch_prover.rs

```diff
@@ -0,0 +1,462 @@
+#[cfg(not(feature = "std"))]
+use alloc::vec;
```

Comment:
```suggestion
```

`cargo check --no-default-features` show this as unused. I guess it's cause in tests it's already imported.